vaadin-login-flow/src/main/java/com/vaadin/flow/component/login/LoginI18n.java

```diff
@@ -41,7 +41,7 @@
         try {
             final JsonFactory JSON_FACTORY = new JreJsonFactory();
             DEFAULT_I18N = JSON_FACTORY.parse(IOUtils
-                .toString(LoginI18n.class.getResource("/i18n.json"),
+                .toString(LoginI18n.class.getResource("i18n.json"),
                     Charset.defaultCharset()));
```

Comment:
This makes parsing platform-dependent, prefer `Charset.forName("UTF-8")` instead since we know that that JSON file is UTF-8.